domain/src/main/java/co/kr/tnt/domain/model/RecordType.kt

```diff
@@ -0,0 +1,22 @@
+package co.kr.tnt.domain.model
+
+sealed class RecordType {
+    sealed class DietType : RecordType() {
+        data object BREAKFAST : DietType()
+        data object LUNCH : DietType()
+        data object DINNER : DietType()
+        data object SNACK : DietType()
+    }
+
+    sealed class ExerciseType : RecordType() {
+        data object UPPER_BODY : ExerciseType()
+        data object LOWER_BODY : ExerciseType()
+        data object BACK : ExerciseType()
+        data object SHOULDER : ExerciseType()
+        data object CARDIO : ExerciseType()
+    }
+
+    sealed class PTSessionType : RecordType() {
```

Comment:
```suggestion
    data class PTSessionType(val sessionCount: Int) : RecordType()
```

위 형태로 만들면, `create` 메소드에서 `sessionCount` 를 `nullable` 하게 설정하지 않아도 될 것 같아요.

항상 메소드나 클래스를 생성할 때 **'특정 상황에서만 사용되는 파라미터' 의 주입**은 피하는게 좋습니다!

Reply:
수정해습니다!! 주의하겠습니다💦